apps/keyboard/js/layouts/emoji.js

```diff
@@ -0,0 +1,274 @@
+Keyboards.emoji = {
+  label: 'Emoji',
+  types: ['text', 'url', 'email'],
+  menuLabel: 'Emoji',
+  pages: [{   // Page 0 - People
+    pageType: 'emoji',
+    emojiKeys: ['😄', '😃', '😀', '😊', '☺', '😉',
```

Comment:
I want to see 

```javascript
sectionKeys: [
   [{ key object } ... ],
   [{ key object } ... ]
]
```

And specify the key should be rendered by `EmojiKeyView` in key object.